lib/scoreboard.rb

```diff
@@ -0,0 +1,34 @@
+class Scoreboard
+  def initialize
+    @rolls = [] # array that will store down number of knocked pins after each roll
+    @current_roll = 0 #to keep track of the current roll. Initalising means it is ready to take information so it 
```

Comment:
This could be maybe a reference to the frame, once we detect that we should move to the next frame. It is not used in the code.